src/e2e-test/java/io/cdap/plugin/pubsub/actions/PubSubActions.java

```diff
@@ -0,0 +1,95 @@
+
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.pubsub.actions;
```

Comment:
This is for the pub/sub sink plugin, not the streaming source. Please rename to make this clear.
Also, we should add tests for the streaming source (can be done in separate PR)

Reply:
Renamed